stacker/builder.py

```diff
@@ -25,17 +24,17 @@ def __str__(self):
         return self.message
 
 
-def get_stack_full_name(cfn_domain, stack_name):
-    return "%s-%s" % (cfn_domain, stack_name)
+def get_stack_full_name(cfn_base, stack_name):
+    return "%s-%s" % (cfn_base, stack_name)
 
 
 def stack_template_key_name(blueprint):
     return "%s-%s.json" % (blueprint.name, blueprint.version)
 
 
-def stack_template_url(cfn_domain, blueprint):
+def stack_template_url(bucket_name, blueprint):
     key_name = stack_template_key_name(blueprint)
-    return "https://s3.amazonaws.com/%s/%s" % (cfn_domain, key_name)
+    return "https://s3.amazonaws.com/%s/%s" % (bucket_name, key_name)
```

Comment:
You don't have to, you can pass the body directly, but if you do you are limited to 51k.  If you use a templateURL you are allowed to have a policy as large as 460k.  In general it's preferred to store them in S3.  We version the templates, so another benefit is that you can see every version you've ever uploaded of a stack in that bucket.

http://docs.aws.amazon.com/AWSCloudFormation/latest/APIReference/API_CreateStack.html (See TemplateBody/TemplateURL)